ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -14,16 +16,19 @@
 
 @elementwise(input_type=[dt.double], output_type=dt.double)
 def add_one(s):
+    assert isinstance(s, (np.ndarray, pd.Series))
```

Comment:
I think we only need to loose the assertion on reduction to be `isinstance(s, (np.ndarray, pd.Series))`

elementwise and analytic only takes pd.Series

Reply:
I can try that out.